### PR TITLE
Provide key size for RSA key

### PR DESCRIPTION
**Pull request by BethanyZhou**
## Description
```
Get-AzKeyVaultKey -VaultName bez-kv0512 -Name test1


Vault/HSM Name : bez-kv0512
Name           : test1
Key Type       : RSA
Key Size       : 2048
Version        : a99fe8b2325f4d1baf19871c3a920ee4
Id             : https://bez-kv0512.vault.azure.net:443/keys/test1/a99fe8b2325f4d1baf19871c3a920ee4
Enabled        : True
Expires        :
Not Before     :
Created        : 5/13/2021 3:25:44 AM
Updated        : 5/13/2021 3:25:44 AM
Recovery Level : Recoverable+Purgeable
Tags           :
```

## Checklist

- [x] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [x] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
        - {Please put the link here}
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

File: src/KeyVault/KeyVault/Helpers/JwkHelper.cs [hunk 1]
```diff
@@ -7,6 +7,21 @@ namespace Microsoft.Azure.Commands.KeyVault.Helpers
 {
     internal static class JwkHelper
     {
+        internal static RSACryptoServiceProvider ConvertToRSAKey(JsonWebKey jwk)
+        {
+            if (jwk == null || !jwk.Kty.Equals("RSA"))
```
Comment (dingmeng-xue) [inline src/KeyVault/KeyVault/Helpers/JwkHelper.cs:12]:
```suggestion
            if (!"RSA".Equals(jwk?.Kty))
```
Is it better?

Comment (BethanyZhou) [reply src/KeyVault/KeyVault/Helpers/JwkHelper.cs:12]: Yes!

File: src/KeyVault/KeyVault/Helpers/JwkHelper.cs [hunk 2]
```diff
@@ -7,6 +7,21 @@ namespace Microsoft.Azure.Commands.KeyVault.Helpers
...
+            {
+                return null;
+            }
+            var csp = new RSACryptoServiceProvider();
+            csp.ImportParameters(new RSAParameters()
```
Comment (dingmeng-xue) [inline src/KeyVault/KeyVault/Helpers/JwkHelper.cs:17]: We need to catch the exception. If there is any error, let's return null. 

https://docs.microsoft.com/en-us/dotnet/api/system.security.cryptography.rsacryptoserviceprovider.importparameters?view=net-5.0#System_Security_Cryptography_RSACryptoServiceProvider_ImportParameters_System_Security_Cryptography_RSAParameters_

**Review by dingmeng-xue — APPROVED** [review]

**Comment by Gill-Bates** [comment]
@dingmeng-xue Any Update on this?

**Comment by dingmeng-xue** [comment]
You can see this PR is merged. So, the feature should be available on May 25. Could you look into it? https://www.powershellgallery.com/packages/Az.KeyVault/3.4.4

**Comment by Gill-Bates** [comment]
@dingmeng-xue: I am using 3.4.5. It seems not working:

`$PSVersionTable`

```
Name                           Value
----                           -----
PSVersion                      7.1.3
PSEdition                      Core
GitCommitId                    7.1.3
OS                             Microsoft Windows 10.0.18363
Platform                       Win32NT
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0…}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0
```

`Get-Module`

```
Script     2.5.2                 Az.Accounts                         {Add-AzEnvironment, Clear-AzContext, Clear-AzDefa…
Script     3.4.5                 Az.KeyVault                         {Add-AzKeyVaultCertificate, Add-AzKeyVaultCertifi…
```

`Get-AzKeyVaultKey -VaultName myKeyVault`
```
Vault/HSM Name : myKeyVault
Name           : saencryptionkey2*****
Version        :
Id             : https://myKeyVault.vault.azure.net:443/keys/saencryptionkey2*****
Enabled        : True
Expires        : 15.12.2021 13:55:09
Not Before     :
Created        : 31.08.2021 13:02:38
Updated        : 31.08.2021 13:02:38
Recovery Level : Recoverable
Tags           :
```

**Pull request merged**